{
  "errorName": "UserNotExistsException",
  "cause": "null",
  "errorMsg": "User with such email doesn't exists!"
}